src/main/java/com/codecool/virtualpub/ui/Display.java

```diff
@@ -99,7 +99,7 @@ public void displayBartenderActions(String bartenderName, String[] actions) {
      */
     public void displayDrinkName(String[] drinkNames) {
         System.out.println("\n" + "Drinks: ");
-        for (int i = 1; i < drinkNames.length; i++) {
+        for (int i = 1; i <= drinkNames.length; i++) {
             System.out.println(i + ": " + drinkNames[i - 1]);
```

Comment:
Every message printing should use the displayMessage function to make it easier if some changes are made to displaying. For example if we decide we want every message to be displayed with yellow text, then we only need to change the displayMessage method and every display will automaticaly change.